Description: Recently, vala-doc started using .svg image files (rather than .png).
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -29,7 +29,7 @@
 	*.css \
 	*.devhelp2 \
 	*.html \
-	*.png \
+	*.svg \
 	$(NULL)
 
 doc_DATA = \
